pixl_dcmd/src/pixl_dcmd/tests/test_main.py

```diff
@@ -0,0 +1,87 @@
+#  Copyright (c) 2022 University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from __future__ import annotations
+
+import pathlib
+
+import pydicom
+import pytest
+import sqlalchemy
+import yaml
+from pydicom.data import get_testdata_files
+
+from core.database import Image
+from pixl_dcmd.main import (
+    apply_tag_scheme,
+    remove_overlays,
+)
+
+
+@pytest.fixture(scope="module")
+def tag_scheme() -> dict:
+    """Read the tag scheme from orthanc raw."""
+    tag_file = (
+        pathlib.Path(__file__).parents[4]
+        / "orthanc/orthanc-anon/plugin/tag-operations.yaml"
+    )
+    return yaml.safe_load(tag_file.read_text())
+
+
+def test_remove_overlay_plane() -> None:
+    """Checks that overlay planes are removed."""
+    fpath = get_testdata_files("MR-SIEMENS-DICOM-WithOverlays.dcm")[0]
+    ds = pydicom.dcmread(fpath)
```

Comment:
Maybe try this instead?
```suggestion
    ds = get_testdata_file("MR-SIEMENS-DICOM-WithOverlays.dcm", read=True, download=True)
```